### PR TITLE
Add unit tests for utility functions in compound prediction

### PR DESCRIPTION
**Pull request by Cidana-Developers**
1. add unit test for utility functions in wedge prediction, including:
   av1_wedge_sign_from_residuals_avx2,
   av1_wedge_compute_delta_squares_avx2,
   av1_wedge_sse_from_residuals_avx2
2. add unit tests for utility functions:
   av1_build_compound_diffwtd_mask_{, d16}avx2,
   aom_blend_a64_mask_avx2/aom_lowbd_blend_a64_d16_mask_avx2

### CODE REVIEW

File: test/WedgeUtilTest.cc [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+TEST_F(WedgeUtilTest, MaskSignTest) {
+    const int loops = 10000;
+    const int int_13bit_max = (1 << 12) - 1;
+    SVTRandom rnd(13, true);             // max residual is 13-bit
+    SVTRandom m_rnd(0, MAX_MASK_VALUE);  // [0, MAX_MASK_VALUE]
+    SVTRandom n_rnd(1, 8191 / 64);       // required by assembly implementation
+    DECLARE_ALIGNED(32, int16_t, ds[MAX_SB_SQUARE]);
+
+    for (int k = 0; k < loops; ++k) {
+        if (k < 4) {  // extreme value test
```
Comment (andrey-norkin) [inline test/WedgeUtilTest.cc]: Why not first test these four values and then continue with the random ones? Then you do not need to make a check for k < 4 every time. Also name loops is not the best name

File: test/WedgeUtilTest.cc [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+// calculate the sse of two prediction combined with mask m
+TEST_F(WedgeUtilTest, SseFromResidualTest) {
+    const int loops = 10000;
+    const int int_13bit_max = (1 << 12) - 1;
+    SVTRandom rnd(13, true);             // max residual is 13-bit
+    SVTRandom m_rnd(0, MAX_MASK_VALUE);  // [0, MAX_MASK_VALUE]
+    SVTRandom n_rnd(1, MAX_SB_SQUARE / 64);
+
+    for (int k = 0; k < loops; ++k) {
+        if (k < 4) {
```
Comment (andrey-norkin) [inline test/WedgeUtilTest.cc]: Same here

Comment (Cidana-Developers) [reply test/WedgeUtilTest.cc]: Modified, please review it again.

File: test/WedgeUtilTest.cc (hunk 1 above)
Comment (Cidana-Developers) [reply test/WedgeUtilTest.cc]: Modified, please review it.

**Comment by 1480c1** [comment]
It seems that the reason the appveyor builds are failing is because over the past 6 months, all of the dlls and exes have accumulated to 50gb within this repo on appveyor's side

**Pull request merged**